.travis.yml

```diff
@@ -7,3 +7,9 @@ language: java
 
 jdk:
   - oraclejdk8
+
+deploy:
+  provider: bintray
+  file: descriptor.json
+  user: vaani
+  key: 315213b83b21bc121e7ff476148a598193db810b
```

Comment:
Again[[1]](http://stackoverflow.com/questions/32356238/issue-using-encrypted-bintray-key-in-travis-yml) this looks like our password is in the open.